src/avx/avx_mixed_radix.rs

```diff
@@ -229,17 +229,13 @@ macro_rules! mixedradix_column_butterflies {
                     let mut columns = [AvxVector::zero(); ROW_COUNT];
                     if partial_remainder == 1 {
                         for i in 0..ROW_COUNT {
-                            columns[i] = AvxArray::<A>::load_partial1_complex(
-                                &buffer,
-                                partial_remainder_base + len_per_row * i,
-                            );
+                            columns[i] = buffer
+                                .load_partial1_complex(partial_remainder_base + len_per_row * i);
                         }
                     } else {
                         for i in 0..ROW_COUNT {
-                            columns[i] = AvxArray::<A>::load_partial2_complex(
-                                &buffer,
-                                partial_remainder_base + len_per_row * i,
-                            );
+                            columns[i] = buffer
+                                .load_partial2_complex(partial_remainder_base + len_per_row * i);
```

Comment:
Nice! I always hated having to manually specify the trait here, this is way nicer

Reply:
Most of these instances were changed already actually! I just found a few more to update